compiler-rt/lib/scudo/standalone/primary64.h

```diff
@@ -601,6 +593,35 @@ template <typename Config> class SizeClassAllocator64 {
     return BlockSize > PageSize;
   }
 
+  void initRegion(RegionInfo *Region, uptr ClassId, MemMapT MemMap,
```

Comment:
Should this be made to be always inline? If it's not used many places, it will probably be inlined anyway though.